pkg/granted/sso.go

```diff
@@ -245,7 +265,3 @@ func mergeSSOProfiles(config *ini.File, prefix string, ssoProfiles []SSOProfile)
 
 	return nil
 }
-
-func normalizeAccountName(accountName string) string {
```

Comment:
Did you have a specific use-case in mind where you want the account name normalised in a different way?